go.mod

```diff
@@ -11,8 +12,8 @@ require (
 	github.com/onsi/gomega v1.19.0
 	github.com/openshift/api v0.0.0-20211028023115-7224b732cc14
 	github.com/openshift/client-go v0.0.0-20210831095141-e19a065e79f7
+	github.com/openshift/custom-resource-status v1.1.2
 	github.com/operator-framework/api v0.14.0
-	github.com/operator-framework/operator-lifecycle-manager v0.20.0
```

Comment:
We use OLM for e2e tests. Will removing this cause issues?

Reply:
I ran go mod tidy and it removed it automatically, which means we are not using it anymore.